content/bn/zero-trust-architecture.md

```diff
@@ -0,0 +1,16 @@
+---
+title: জিরো ট্রাস্ট আর্কিটেকচার (Zero Trust Architecture)
+status: Completed
+category: নিরাপত্তা
+---
+
+## এটি কি
+জিরো ট্রাস্ট আর্কিটেকচার আইটি সিস্টেমের ডিজাইন এবং বাস্তবায়নের জন্য একটি নির্ধারিত পন্থা যেখানে 'ট্রাস্ট'(বিশ্বাস) সম্পূর্ণরূপে উপেক্ষা করা হয়। এই পন্থার মূল নীতি হল "বিশ্বাস নয়,সর্বদা যাচাই করা",ডিভাইস বা সিস্টেম,সিস্টেমের অন্যান্য অংশের সাথে যোগাযোগ করার সময় সর্বদা নিজেকে প্রথমে যাচাই করে। বর্তমানে অনেক নেটওয়ার্কে,কর্পোরেট নেটওয়ার্কের মধ্যে থাকা সিস্টেম এবং ডিভাইসগুলি অবাধে একটি অন্যটির সাথে যোগাযোগ করতে পারে কারণ সিস্টেম এবং ডিভাইসগুলি  কর্পোরেট নেটওয়ার্ক পরিধির বিশ্বস্ত সীমার মধ্যে আবদ্ধ থাকে। অন্যদিকে জিরো ট্রাস্ট আর্কিটেকচার বিপরীত পদ্ধতি অবলম্বন করে যেখানে নেটওয়ার্ক পরিধির মধ্যেই সিস্টেমের অংশগুলি কোন যোগাযোগ স্থাপনের জন্য প্রথমে যাচাইকরণ পর্যায় অতিক্রম করে।
+
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+চিরাচরিত বিশ্বাস ভিত্তিক পদ্ধতি অনুসারে যেখানে সিস্টেম এবং ডিভাইসগুলি কর্পোরেট নেটওয়ার্কের পরিধির মধ্যে থাকে,অনুমান করা হয় যে,যেহেতু বিশ্বাস আছে,সেহেতু কোন সমস্যা নেই। অবশ্য,জিরো ট্রাস্ট আর্কিটেকচার স্বীকার করে যে বিশ্বাস(ট্রাস্ট) একটি দুর্বলতা। এখন যদি একটি ঘটনা ঘটে যেখানে,এক আক্রমণকারী একটি বিশ্বস্ত ডিভাইসে রাখা বিশ্বাস এবং দেওয়া অ্যাক্সেস স্তরের উপর নির্ভর করে ডিভাইসটির অ্যাক্সেস পায়,তাহলে সেই আক্রমণকারী "বিশ্বস্ত" নেটওয়ার্ক পরিধির মধ্যে থাকায় সিস্টেমটি এখন আক্রমণের জন্য ঝুঁকিপূর্ণ এবং আক্রমণকারী পুরো সিস্টেম জুড়ে পার্শ্বীয়ভাবে জায়গা বদলাতে (move laterally) সক্ষম। একটি জিরো ট্রাস্ট আর্কিটেকচারে,বিশ্বাসকে উপেক্ষা করা হয়,তাই আক্রমণের পৃষ্ঠ হ্রাস পায় ফলে,একজন আক্রমণকারীকে সিস্টেমের আরো অভ্যন্তরে যাওয়ার আগে যাচাইকরণ করাতে বাধ্য করা হয়।
```

Comment:
Can you please create an easier sentence instead of this?
""এখন যদি একটি ঘটনা ঘটে যেখানে,এক আক্রমণকারী একটি বিশ্বস্ত ডিভাইসে রাখা বিশ্বাস এবং দেওয়া অ্যাক্সেস স্তরের উপর নির্ভর করে ডিভাইসটির অ্যাক্সেস পায়,তাহলে সেই আক্রমণকারী "বিশ্বস্ত" নেটওয়ার্ক পরিধির মধ্যে থাকায় সিস্টেমটি এখন আক্রমণের জন্য ঝুঁকিপূর্ণ এবং আক্রমণকারী পুরো সিস্টেম জুড়ে পার্শ্বীয়ভাবে জায়গা বদলাতে (move laterally) সক্ষম""

Reply:
okay